src/GameOptions/ui/CreditsModal.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Modal } from "../../ui/React/Modal";
+import { Typography, Link, Button } from "@mui/material";
+
+interface IProps {
+  open: boolean;
+  onClose: () => void;
+}
+
+export function CreditsModal(props: IProps): React.ReactElement {
+  const leadDevs = `danielyxie
+Olivier Gagnon
+@Snarling
+`;
+
+  const currentMaintainer = `@Snarling`;
+
+  const contributorsURL = `https://github.com/bitburner-official/bitburner-src/graphs/contributors`;
+  const contributorsMessage = `Visit GitHub to see all contributors
+or to participate yourself`;
+
+  return (
+    <Modal open={props.open} onClose={props.onClose}>
```

Comment:
If possible, probably better to avoid duplicating `align="center"` in the following lines - I wasn't able to do that with sx , style, etc. in line 23.